src/inspect.js

```diff
@@ -47,11 +48,8 @@ export function silenceError (p) {
 	p._runAction(silencer)
 }
 
-const silencer = {
-	fulfilled () {},
-	rejected: setHandled
-}
-
-function setHandled (rejected) {
-	rejected._state |= HANDLED
+const silencer = new Action(null)
```

Comment:
It may.  We may already be in hidden class transition land, since Action subclasses tend to have slightly different shapes.  I agree: since we don't really know the effect, it seems like the simplest thing is not to change silencer from what it is in master.